components/Starknet/modules/starknet_versions/pages/version_notes.adoc

```diff
@@ -13,7 +13,6 @@ Within Starknet's deployment pipeline there are separate and distinct networks t
 |*Environment* |*Starknet version*|*Sierra version*|*Cairo version*
 |Mainnet|0.12.2|1.3.0|2.0.0 - 2.3.0
 |Goerli Testnet 1|0.12.2|1.3.0|2.0.0 - 2.3.0
-|Goerli Testnet 2|0.12.2|1.3.0|2.0.0 - 2.3.0
```

Comment:
@odednaor Deprecated means that it's still supported, but will be removed in the future, so we need to keep this in the docs but mark it as deprecated.

If you mean that it is now removed and no longer supported, that's different. Please clarify. This impacts other pages as well, such as https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/transactions/#chain-id.

Reply:
So let's not remove the line from the table and just mention that it's deprecated.